examples/src/main/python/mllib/binary_classification_metrics_example.py

```diff
@@ -27,7 +27,7 @@
 
 if __name__ == "__main__":
     sc = SparkContext(appName="BinaryClassificationMetricsExample")
```

Comment:
you didn't fix this one, is it because it didn't use `SQLContext` and we actually needed the underlying `SparkContext`?